components/comments/commentItem.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
```

Comment:
I know some of our components file names are `camelCase` but that's because that's the name of the content type (it use the content type variable) the convention for React is `SnakeCase`.

Reply:
weird, I changed this but for some reasons it does not get reflected here!